ecs-cli/modules/cli/image/image_app_test.go

```diff
@@ -38,8 +38,8 @@ const (
 	repositoryWithSlash = "hi/repo"
 	tag                 = "tag-v0.1.0"
 	image               = repository + ":" + tag
-	registry            = "registry"
-	registryID          = "123456789"
+	registry            = "https://" + registryID + ".dkr.ecr.us-west-2.amazonaws.com"
```

Comment:
Does this mean that the https is always required when using fips? That seems kind of confusing and inconvenient for customers- is there a way we can make it so that its not required?